plugins/woocommerce-admin/client/layout/store-alerts/index.js

```diff
@@ -81,8 +81,9 @@ export class StoreAlerts extends Component {
 
 						// navigate to previous alert to avoid an out of bounds error in case it's the last alert from the array
 						this.previousAlert();
-
-						await triggerNoteAction( alert.id, action.id );
+						if ( alert.id && action.id ) {
```

Comment:
IIUC, when the `ids` are undefined, clicking the button will do nothing. Do we need to display an error notice if this check is added here?